### DISCUSSION TITLE
Test discussion

### DISCUSSION THREAD

**Discussion by MIKEX818** [Cyfrin/foundry-full-course-cu#1831] [General]
I am receiving this error while compiling:

```
ParserError: Expected identifier but got '('
  --> FundMe.sol:43:12:
   |
43 |     fallbac() external payable {
   |            ^
```

Here is my code:

```solidity
    fallbac() external payable {
        fund();
    }
```

Can someone help me figure what's the issue is?

**Comment by Ifra001** [comment]
I think its because you spelled ```fallback()``` wrong. Try adding a k at the end of fallbac

**Comment by urosognjenovic** [comment]
The function you're trying to implement is ```fallback```. It is a unique function in Solidity as it doesn't require the ```function``` keyword. Therefore, spelling it wrong and without the ```function``` keyword will throw an error.

You just need:
```
fallback() external payable {
    fund();
}
```

**Comment by EngrPips** [comment]
Good job here guys.
